components/search-tab-group/search-tab-group.jsx

```diff
@@ -84,7 +88,7 @@ class SearchTabGroup extends React.Component {
     return SearchTab(
```

Comment:
I know this was in place already, but what is going on here? The syntax on the line you changed is messing with my head.

Reply:
Oh. `SearchTab` returns a function. That makes more sense.